test/Microsoft.AspNet.Mvc.FunctionalTests/XmlSerializerInputFormatterTests.cs

```diff
@@ -46,9 +46,9 @@ public class XmlSerializerInputFormatterTests
             var sampleInputInt = 10;
             var input = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
                 "<DummyClas xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\" " +
-                "i:type=\"DerivedDummyClass\" xmlns=\"http://schemas.datacontract.org/2004/07/XmlSerializerWebSite\">" +
+                "i:type=\"DerivedDummyClass\" xmlns=\"http://schemas.datacontract.org/2004/07/XmlFormattersWebSite\">" +
                 "<SampleInt>" + sampleInputInt.ToString() + "</SampleInt></DummyClass>";
-            var content = new StringContent(input, Encoding.UTF8, "application/xml");
+            var content = new StringContent(input, Encoding.UTF8, "application/xml-xmlser");
```

Comment:
a small comment in the startup stating so would help. It is not obvious who is not familiar with xml  stuff :smile: